augly/tests/image_tests/transforms_unit_test.py

```diff
@@ -201,6 +201,9 @@ def test_Sharpen(self):
 
     def test_ShufflePixels(self):
         self.evaluate_class(imaugs.ShufflePixels(factor=0.5), fname="shuffle_pixels")
+    
+    def test_Skew(self):
+        self.evaluate_class(imaugs.Skew(skew_factor=0.5, axis=0), fname="skew")
```

Comment:
```suggestion
        self.evaluate_class(imaugs.Skew(), fname="skew")
```
These are already the default values of the params, so unnecessary.